lib/ansible/modules/notification/logentries_msg.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: logentries_msg
+version_added: "2.2"
+short_description: Send a message to logentries.
+description:
+   - Send a message to logentries
+requirements:
+  - "python >= 2.6"
+options:
+  token:
+    description:
+      - Log token.
+    required: true
+    version_added: "2.1"
+  msg:
+    description:
+      - The message body.
+    required: true
+    default: null
```

Comment:
If a parameter is required, it does not have a default. So please remove the `default: null`.